xclim/indices/_conversion.py

```diff
@@ -1000,20 +1000,20 @@ def longwave_upwelling_radiation_from_net_downwelling(
     Parameters
     ----------
     rls : xr.DataArray
-        Surface net thermal radiation [W m-2].
+        Surface net thermal radiation.
     rlds : xr.DataArray
-        Surface downwelling thermal radiation [W m-2].
+        Surface downwelling thermal radiation.
 
     Returns
     -------
-    xr.DataArray, [W m-2]
+    xr.DataArray, [same as inputs]
```

Comment:
To imitate what `daily_temperature_range` which computes `tasmax - tasmin`:
```suggestion
    xr.DataArray, [same units as rlds]
```